model/session.go

```diff
@@ -140,7 +143,37 @@ func (me *Session) GetTeamByTeamId(teamId string) *TeamMember {
 }
 
 func (me *Session) IsMobileApp() bool {
-	return len(me.DeviceId) > 0
+	return len(me.DeviceId) > 0 || me.IsMobile()
+}
+
+func (me *Session) IsMobile() bool {
+	val, ok := me.Props[USER_AUTH_SERVICE_IS_MOBILE]
+	if !ok {
+		return false
+	}
+	isMobile, err := strconv.ParseBool(val)
+	if err != nil {
+		mlog.Debug(err.Error())
```

Comment:
These should be logged as `mlog.Error()`. Production systems normally will not have Debug logging turned on. If these errors are encountered, we want them to be logged. Also, should add a description for the error rather than just the error message.

Reply:
Good point. I knew about this but didn't really take it into consideration.